packages/docs/index.js

```diff
@@ -65,7 +66,9 @@ const main = async () => {
 
   const outputs = await Promise.all(
     pages.map(([stem, doc]) =>
-      markdownPlugin(doc, path.join(outPath, stem + ".md"))
+      Object.keys(doc).map((scope) =>
```

Comment:
I did this earlier so the output can include docs for multiple scopes into a single file.

This is important sometimes. For example:

* the `rule` scope (which has `bind` and `return` doesn't need it's own unique doc page, but lives inside the `std` doc): https://github.com/windsorio/browse/wiki/API-Reference. The docs there even use `@link` between the scopes since they are tightly coupled, so it's imperative that they are in the same file

* the `page` and `browser` scopes should be in a single file, maybe called `web` since they are coupled. Or, they could be in their own files, but should probably atleast be in their own folder since they are tightly coupled

I think what I did was "better" (it's still opinionated  since we assume files are the final output, but really we're copy-pasting things to github wiki, and ideally we can abstract this out like JSDoc/JavaDocs etc. does so it simply spits out an IR, and then there's a different plugin that produces HTML/Markdown/.txt files etc.

I think you're going a step back from that by limiting things to one scope per file

Reply:
IIRC I did this because I wanted the default behavior to be that the filename was the scope name in the case that the user doesn't provide an @name argument. Using this method the default name for Math was 'main' because it's in the main file. I'll fix it so that both functionalities stay intact.

Reply:
Makes sense